crates/driver/src/domain/liquidity/fetcher.rs

```diff
@@ -1,17 +0,0 @@
-use crate::domain::{competition::auction, liquidity};
```

Comment:
This technically moved to `crates/driver/src/domain/liquidity/fetcher/mod.rs`, GitHub just doesn't show it that way.